src/index.html

```diff
@@ -5,10 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home" class="logo__link">
+          <img
+          class="logo__image"
+          src="images/logo.png"
+          alt="logo"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="is-active nav__link">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">SMARTPHONES</a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#computers"
+            data-qa="hover"
+            class="nav__link"
+            >LAPTOPS & COMPUTERS</a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">GADGETS</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">TABLETS</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">PHOTO</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">VIDEO</a>
```

Comment:
in requirements we said that we want to uppercase list item using CSS instead of HTML